### PR TITLE
Update README.md to catch recent progress on Ansible integration

### PR DESCRIPTION
**Pull request by AstroProfundis**
- pull code base to latest `pingcap/master`
 - update `insight.py -h` help output
 - update note about ansible integration

### CODE REVIEW

**Comment by AstroProfundis** [comment]
@ethercflow PTAL.
The docs about `vmtouch`, `blktrace`, etc. are not added yet, you can add them by forking from branch `docs` and them merge them back.

**Comment by AstroProfundis** [comment]
@CaitinChen PTAL

File: README.md [hunk 1]
```diff
@@ -47,11 +64,23 @@ optional arguments:
...
+                        server will be used by default.
+  -c, --compress        Compress all output files to a tarball and delete
+                        temporary files, disabled by default.
   -p, --perf            Collect trace info using perf. Disabled by default.
   --pid PID             PID of process to run perf on. If `-p`/`--perf` is not
                         set, this value will not take effect. Multiple PIDs
                         can be set by using more than one `--pid` argument.
                         `None` by default which means the whole system.
+  --proc-listen-port PROC_LISTEN_PORT
+                        Collect perf data of process that listen on given
```
Comment (CaitinChen) [inline README.md]: listen -> listens

File: README.md [hunk 2]
```diff
@@ -60,15 +89,67 @@ optional arguments:
                         Event sampling frequency of perf-record, in Hz.
   --perf-time PERF_TIME
                         Time period of perf recording, in seconds.
+  --perf-archive        Run `perf archive` after collecting data, useful when
+                        reading data on another machine, may cause large
+                        output file size. Disabled by default.
   -l, --log             Collect log files in output. PD/TiDB/TiKV logs are
-                        included by default.
+                        included by default if no other argument given.
```
Comment (CaitinChen) [inline README.md]: -> included by default if no other argument **is** given.

File: README.md [hunk 3]
```diff
@@ -60,15 +89,67 @@ optional arguments:
...
+                        value will be ignored.
+  --log-prefix LOG_PREFIX
+                        The prefix of log files, will be the directory name of
+                        all logs, will be in the name of output tarball. If
+                        `--log-auto` is set, this value will be ignored.
+  --log-retention LOG_RETENTION
+                        The time of log retention, any log files older than
+                        given time period from current time will not be
+                        included. Value should be a number of hour(s) in
+                        positive interger. `0` by default and means no time
```
Comment (CaitinChen) [inline README.md]: -> positive interger. `0` by default means no time

File: README.md [hunk 4]
```diff
@@ -94,8 +175,10 @@ The `tidb-insight` project is designed to intergrate with [tidb-ansible](https:/
 
 > All collecting features will be disabled by default after the development of intergration is finished, so it's highly recommended to specify all the arguments needed when using `tidb-insight` now, even if the feature is currently enabled by default.
 
-TO DO: Add documents/links to documents of using `tidb-insight` with `tidb-ansible`.
+The `tidb-insight` tarball is included in `tidb-ansible` as part of playbook `collect_diagnosis`, it deploies the pre-build `tidb-insight` dependencies and tools, as well as scripts to target server. Several tasks can be called directly from `ansible-playbook` such as collecting log files of all deployed TiDB/TiKV/PD instances (While `--log-auto` of `tidb-insight` only collects log files of running processes), and copy them back to the controller server in `.tar.gz` format.
```
Comment (CaitinChen) [inline README.md]: The `tidb-insight` tarball is included in `tidb-ansible` as part of playbook `collect_diagnosis`, it **deploys** the pre-build `tidb-insight` dependencies and tools, as well as scripts to **the** target server. Several tasks can be called directly from `ansible-playbook` such as collecting log files of all deployed TiDB/TiKV/PD instances (While `--log-auto` of `tidb-insight` only collects log files of running processes), and copy them back to the controller server in `.tar.gz` format.

File: README.md [hunk 5]
```diff
@@ -94,8 +175,10 @@ The `tidb-insight` project is designed to intergrate with [tidb-ansible](https:/
...
+
+There might be features that are not yet implemented or enabled in `tidb-ansible`, that users can login to target server to call `tidb-insight` by hand. The package is located at `deploy/scripts/tidb-insight` on target server.
```
Comment (CaitinChen) [inline README.md]: -> There might be features that are not yet implemented or enabled in `tidb-ansible`. Users can login to the target server to call `tidb-insight` and execute these features by hand. The package is located at `deploy/scripts/tidb-insight` on the target server.

File: measurement/util.py [hunk 6]
```diff
@@ -97,10 +98,10 @@ def parse_insight_opts():
     parser.add_argument("--perf-time", type=int, action="store", default=None,
                         help="Time period of perf recording, in seconds.")
     parser.add_argument("--perf-archive", action="store_true", default=False,
-                        help="Run `perf archive` after collecting data, useful when reading data on another machine. Disabled by default.")
+                        help="Run `perf archive` after collecting data, useful when reading data on another machine, may cause large output file size. Disabled by default.")
 
     parser.add_argument("-l", "--log", action="store_true", default=False,
-                        help="Collect log files in output. PD/TiDB/TiKV logs are included by default.")
+                        help="Collect log files in output. PD/TiDB/TiKV logs are included by default if no other argument given.")
```
Comment (CaitinChen) [inline measurement/util.py]: ->  help="Collect log files in output. PD/TiDB/TiKV logs are included by default if no other argument **is** given.")

**Pull request closed without merging**